config/v1/types_cluster_version.go

```diff
@@ -551,8 +551,82 @@ type Update struct {
 	//
 	// +optional
 	Force bool `json:"force"`
+
+	// schedule indicates when the update should be started. If this isn't
+	// explicitly set then the update will be started immediately after the
+	// required images have been downloaded.
+	//
+	// +optional
+	Schedule UpdateSchedule `json:"schedule,omitempty"`
+
+	// imageSource indicates what is the source for the images required for
+	// the update. If this isn't explicitly set then the images will be
+	// downloaded from their corresponding registry servers.
+	//
+	// +optional
+	ImageSource UpdateImageSource `json:"imageSource,omitempty"`
```

Comment:
Please remove `Schedule UpdateSchedule `json:"schedule,omitempty"` from this PR as it is not part of the parent enhancement

Reply:
It is part of the parent enhancement. It used to be a `hold` boolean flag to indicate if the upgrade has to be performed immediately after the images are pre-loaded or if it should wait for a later time. That functionality was suggested by @williamcaban during the review of the draft of the enhancement.